bin/specextract

```diff
@@ -609,9 +463,9 @@ def set_badpix(evtfile, bpixfile, instrument, verbose):
 
         finally:
             if os.path.isfile(bpixfile) and getattr(ardlib,bpixpar.replace("-","_")).rstrip("[BADPIX]") == bpixfile:
```

Comment:
I think this is better written as

    if not os.path.isfile(bpixfile) or getattr(...) != bpixfile:
        raise IOError(...)

as it is a bit clearer what leads to an error.